src/edu/washington/escience/myria/daemon/MyriaDriver.java

```diff
@@ -590,6 +591,17 @@ private void setJVMOptions(final AllocatedEvaluator evaluator, final boolean isM
                         MyriaGlobalConfigurationModule.WorkerJvmHeapSizeMaxGB.class)));
     final Set<String> jvmOptions =
         globalConfInjector.getNamedInstance(MyriaGlobalConfigurationModule.JvmOptions.class);
+    if (workerConf != null && jvmOptions.contains("-ElasticMem")) {
```

Comment:
What if someone specifies `-ElasticMem` on a standard JVM?

Reply:
Then the JVM would crash with something like: this JVM arg is unknown.